examples/rag_agent/agent/agent.py

```
@@ -0,0 +1,318 @@
+                url=self.qdrant_url,
+                api_key=self.qdrant_api_key
+            )
+
+            # Test connection
+            client.get_collections()
+            vector_size = 1536  # text-embedding-3-small dimensionality
+
+            for db_type, config in COLLECTIONS.items():
+                try:
+                    client.get_collection(config.collection_name)
+                except Exception:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)